QStore4S/src/test/java/edu/asu/qstore4s/message/MessageTest.java

```diff
@@ -9,7 +9,7 @@
 
 public class MessageTest {
     @Test
-    public void testToString() {
+    public void test_toString_success() {
```

Comment:
there should be different test methods for each scenario tested; otherwise, if json creation fails but not XML isn't not obvious right away just by looking at which test failed.